licenses/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Just a reminder to include all files in the recipe otherwise they won't end up in the feedstock. These got left in staged-recipes. Dropping them in PR ( https://github.com/conda-forge/staged-recipes/pull/15063 ). If you need them, please make sure to add them to the feedstock. Thanks!